plugins/pulp_rpm/plugins/migrations/0028_standard_storage_path.py

```diff
@@ -0,0 +1,124 @@
+from pulp.server.db import connection
+
+from pulp.plugins.migration.standard_storage_path import Migration, Plan
+
+
+def migrate(*args, **kwargs):
+    """
+    Migrate content units to use the standard storage path introduced in pulp 2.8.
+    """
+    migration = Migration()
+    migration.add(RPM())
+    migration.add(SRPM())
+    migration.add(DRPM())
+    migration.add(Distribution())
+    migration.add(YumMetadata())
+    migration.add(ISO())
+    migration()
+
+
+class Package(Plan):
+    """
+    The (base) migration plan for package units.
+    """
+
+    key_fields = (
```

Comment:
Actually, looking at other plugin PRs, it looks like subclasses of ``Plan`` only override ``__init__`` and pass in different values to the superclass. What if instead of making subclasses, those are all just instances of this class?

```
package_key_fields = ['some', 'fields']
rpm_migration = Plan(connection.get_collection('units_rpm'), package_key_fields)
drpm_migration = Plan(connection.get_collection('units_drpm'), package_key_fields)
```

and so on.